### DISCUSSION TITLE
Missing PhysXGpu_64.dll

### DISCUSSION THREAD

**Discussion by kcrif** [NVIDIA-Omniverse/PhysX#343] [Q&A] [answered]
As mentioned in https://github.com/microsoft/vcpkg/pull/41216, the required DLL PhysXGpu_64.dll is missing.
Is there a download link for PhysXGpu_64.dll or its own git repository?

**Comment by vreutskyy** [comment]
Maybe this?
https://github.com/NVIDIA-Omniverse/PhysX/releases/tag/106.1-physx-5.4.2

**Comment by kcrif** [comment]
Nope, no PhysXGpu_64.dll within the archive

**Comment by vreutskyy** [comment]
Of course there is
![image](https://github.com/user-attachments/assets/484ccf18-5dcc-40e3-8407-4a8bedc1be7e)

**Comment by kcrif** [comment]
Sorry, at least in the download Archive there is no bin folder 
![image](https://github.com/user-attachments/assets/f9b6eefd-8d5f-420a-8239-4f4e72d4c220)

**Comment by vreutskyy** [comment, marked as answer]
Aa, you're right. You need to run generate_projects.but. It will download PhysXGpu_64.dll for you.

**Reply by FrankXie05** [reply]
Thank you very much for your answer. Does this `PhysXGpu_64.dll` must need to run script `generate_projects.bat` to be built? Thanks also to @kcrif  for his help. :)

**Reply by vreutskyy** [reply]
`generate_projects.bat` generates projects (obviously :)) and downloads `PhysXGpu_64.dll/.so` fitting the target platform. yes, to build PhysX, the projects must be generated first.

**Comment by kcrif** [comment]
Ok, thanks, Maybe you can provide a hint for the Microsoft vcpgk Team also.
